documentation/src/docs/beginner_dev_guide.md

```diff
@@ -0,0 +1,199 @@
+<!--
+Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+# Pravega - Creating Your First Application
+
+Learn how to create a Hello World Pravega app. This guide covers:
+
+* Starting Pravega standalone.
```

Comment:
Perhaps, with mkdocs items like these require a blank line in between.

Reply:
added it.